draft-packages/title-block-zen/docs/TitleBlockZen.stories.tsx

```diff
@@ -55,58 +56,61 @@ const SECONDARY_ACTIONS = [
   },
 ]
 
-export const Default = () => (
+const DefaultTemplate = args => (
   <OffsetPadding>
-    <TitleBlockZen
-      title="Page title"
-      surveyStatus={{ text: "Live", status: "live" }}
-      primaryAction={{
-        label: "Primary link",
-        icon: addIcon,
-        disabled: true,
-        href: "#",
-      }}
-      defaultAction={{
-        label: "Default link",
-        href: "#",
-      }}
-      secondaryActions={SECONDARY_ACTIONS}
-      secondaryOverflowMenuItems={[
-        {
-          action: () => {
-            alert("test")
-          },
-          label: "Overflow action 1",
-          icon: starIcon,
-        },
-        {
-          action: "#",
-          label: "Overflow link 1",
-          icon: starIcon,
-        },
-      ]}
-      handleHamburgerClick={() => {
-        alert("Hamburger clicked")
-      }}
-      breadcrumb={{
-        path: "#",
-        text: "Back to home",
-        handleClick: event => {
-          alert("breadcrumb clicked!")
-        },
-      }}
-      navigationTabs={[
-        <NavigationTab text="Label" href="#" active />,
-        <NavigationTab text="Label" href="#" />,
-        <NavigationTab text="Label" href="#" />,
-        <NavigationTab text="Label" href="#" />,
-        <NavigationTab text="Label" href="#" />,
-        <NavigationTab text="Label" href="#" />,
-      ]}
-    />
+    <TitleBlockZen {...args} />
   </OffsetPadding>
 )
 
+export const Default: Story<Args> = DefaultTemplate.bind({})
+Default.args = {
```

Comment:
I haven't worked much with Storybook yet to comment on this. Seems a bit strange how we're defining fields here rather than when `Default` is declared - though that may be the Storybook standard.

Reply:
I did this way so we can tweak the props live on the UI. 
https://storybook.js.org/docs/react/writing-stories/introduction#using-args

Reply:
Ah, nice. I'll have to explore Storybook more.